setup/baseConfig.yaml

```diff
@@ -62,8 +62,8 @@ libfuse:
   negative-entry-expiration-sec: <time kernel can cache attributes of non existent paths (in sec). Default - 120 sec>
   fuse-trace: true|false <enable libfuse api trace logs for debugging>
   extension: <physical path to extension library>
-  disable-writeback-cache: true|false <disallow libfuse to buffer write requests if you must strictly open files in O_WRONLY or O_APPEND mode. alternatively, you can ignore-open-flags.>
-  ignore-open-flags: true|false <ignore the append and write only flag since O_APPEND and O_WRONLY is not supported with writeback caching. alternatively, you can disable-writeback-cache.>
+  disable-writeback-cache: true|false <disallow libfuse to buffer write requests if you must strictly open files in O_WRONLY or O_APPEND mode. alternatively, you can set ignore-open-flags.>
+  ignore-open-flags: true|false <ignore the append and write only flag since O_APPEND and O_WRONLY is not supported with writeback caching. alternatively, you can disable-writeback-cache. Default value is true>
```

Comment:
so we're not effectively always ignoring the flags that come in with the open call, do you remember what the reasoning was for why we had to introduce this?

Reply:
This has mainly to do with the writeback caching in fuse3. When enabled (which is by default), it causes append/write operations to fail. So, the workaround this is to set the `disable-writeback-cache: true` in config so that libfuse doesn't buffer the write requests.
Alternate option is to set `ignore-open-flags: true` where we ignore the append and write only flags. Refer [this](https://github.com/Azure/azure-storage-fuse/blob/sourav/v2/ignoreOpenFlags/component/libfuse/libfuse_handler.go#L625).

Reply:
Just to share more context on why we added and now why we are changing the default value
Bydefault on fuse3 we enable write-back-cache at kernel level. To populate kernel cache, kernel issues a read command on your file. Now assume you opened a file handle in WRONLY or APPEND mode, which means handle does not have read permissions. When kernel tries to read file using such handle the read operation fails and in turn users Open operation returns back with error. To fix this (as write-back-cache boosts performance) we added a flag to ignore the open flag. This means even if user opens a file in write/append mode we always add read permissions as well.
So if you are not able to open a file in write only or append only mode but want to use cache you need to now give ignore-open-flags=true always. Otherwise disable the write-back cache. We made this a configurable flag because if user goes and checks the permission on the handle it will not be the same as he expects (as we added read permissions). Just to make the expecation right we ask user to explicitly set this flag.
Issues comes from CSI driver which mostly goes with default config. write-back-cache enable and ignore-open-flag set to false. Which means append file or write only open file are bound to fail. And not just CSI driver but any other user by default will hit failure with this default setting.
To resolve this we are now turning ignore-open-flags always true.

Reply:
@vibhansa-msft perfect I added this comment in the pr description for future referenece